Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import csv\r\nimport pandas as pd\r\n\r\n# df = pd.read_csv('soc-sign-bitcoinotc.csv')\r\n# print(df.to_string())\r\n\r\nedgeDict = {}\r\nrevEdgeDict = {}\r\n# edgeDict[\"1\"]=5\r\n# print(edgeDict[\"1\"])\r\ni = 0\r\nwith open(\"soc-sign-bitcoinotc.csv\", 'r') as data:\r\n    for line in csv.reader(data):\r\n        if i == 0:\r\n            i += 1\r\n            continue\r\n        if int(line[0]) in edgeDict.keys():\r\n            edgeDict[int(line[0])].append(line[1])\r\n        else:\r\n            edgeDict[int(line[0])] = [line[1]]\r\n\r\nprint(edgeDict)\r\n\r\n\r\ndef load_graph(path):\r\n    return '1'\r\n\r\n\r\ndef calculate_page_rank(beta, epcil, maxIterations):\r\n    return '1'\r\n\r\n\r\ndef get_PageRank(node_name):\r\n    return '1'\r\n\r\n\r\ndef get_top_PageRank(n):\r\n    return '1'\r\n\r\n\r\ndef get_all_PageRank():\r\n    return 1\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision bd285dc8cee7ba9e44429a93d76a5bbcddbae725)
+++ b/main.py	(date 1668238368953)
@@ -26,7 +26,8 @@
     return '1'
 
 
-def calculate_page_rank(beta, epcil, maxIterations):
+def calculate_page_rank(beta=0.85, epcil=0.001, maxIterations=20):
+    n = 
     return '1'
 
 
